circuitpython_typing/displayio.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 Tim Cocks for Adafruit Industries
+#
+# SPDX-License-Identifier: MIT
+
+"""
+`circuitpython_typing.displayio`
+================================================================================
+
+Type annotation definitions used for displayio related components.
+
+* Author(s): Tim Cocks
+"""
+
+from typing import TypeAlias, Union
+
+from busdisplay import BusDisplay
+from epaperdisplay import EPaperDisplay
+from framebufferio import FramebufferDisplay
+
+AnyDisplay: TypeAlias = Union[BusDisplay | EPaperDisplay | FramebufferDisplay]
```

Comment:
I think you mean `Union[X, Y, Z]` and not `Union[X | Y| Z]`

Reply:
yep, thank you!